website/docs/reference/node-selection/state-comparison-caveats.md

```diff
@@ -60,7 +60,7 @@ models:
 
 </File>
 
-Of course, if the raw Jinja expression is modified, it will be marked as a modification.
+Of course, if the raw Jinja expression is modified, it will be marked as a modification unless you use the `state_modified_compare_more_unrendered` behavior flag to `True`. Read more about this flag in [Behavior changes](/reference/global-configs/behavior-changes#behavior-change-flags)
```

Comment:
```suggestion
Of course, if the raw Jinja expression is modified, it will be marked as a modification unless you set the `state_modified_compare_more_unrendered` behavior flag to `True`. Learn more about [Behavior flags](/reference/global-configs/behavior-changes#behavior-change-flags).
```